.cbmc-batch/include/cipher_openssl.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#ifndef CIPHER_OPENSSL_H
+#define CIPHER_OPENSSL_H
+
+#include <aws/cryptosdk/private/cipher.h>
+
+struct aws_cryptosdk_sig_ctx {
+    struct aws_allocator *alloc;
+    const struct aws_cryptosdk_alg_properties *props;
+    EC_KEY *keypair;
+    EVP_PKEY *pkey;
+    EVP_MD_CTX *ctx;
+    bool is_sign;
+};
+
+struct aws_cryptosdk_md_context {
```

Comment:
I assume you're overriding the definitions from the ESDK here?  What are the key changes?